build/snap/snapcraft.yaml

```diff
@@ -0,0 +1,55 @@
+name: eksctl
+summary: CLI tool.
+description:| a CLI for Amazon EKS
+  
+adopt-info:eksctl
+
+grade: stable
+confinement: strict
+
+architectures:
+  - build-on: i386
+  - build-on: amd64
+  - build-on: armhf
+  - build-on: arm64
+
+apps:
+  eksctl:
+    command: bin/eksctl
+    plugs:
+      - home
+      - network
+      - docker
+      - removable-media
+
+parts:
+  eksctl:
```

Comment:
It doesn't look like valid YAML to me, indentation of the object below this this is off by one level.